src/js/lib/ajax.js

```diff
@@ -101,6 +103,8 @@ var ajax = function(opt, success, failure) {
       try {
         if (opt.type === 'json') {
           body = JSON.parse(body);
+        } else if (op.type === 'xml') {
```

Comment:
Should be `opt.type`. It must be working because of the fallback logic (especially in a try). Sorry about that, the try catch is for guarding against bad servers. Could you test if `body = req.responseXML` works for your use case without the typo?